HW04/tim_bulat/lib/person.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require_relative 'notification'
+
+class Person
+  include Notification
```

Comment:
I think we don't need to create separate module if we only use it in one place, it just increases comlexity. I would prefer to add directly here
```suggestion

```

Reply:
Isn't it was part of homework to create notification module or class?

Reply:
Okay, just for the future